content/docs/command-reference/remote/add.md

```diff
@@ -140,9 +140,19 @@ $ dvc remote modify myremote endpointurl \
                     https://object-storage.example.com
 ```
 
-> See `dvc remote modify` for a full list of S3 API parameters.
```

Comment:
This doesn't give useful information, so I've removed this.

```console
ERROR: the following arguments are required: name, option
usage: dvc remote modify [-h] [--global | --system | --local] [-q | -v] [-u]
                         name option [value]

Modify the configuration of a data remote.
Documentation: <https://man.dvc.org/remote/modify>

positional arguments:
  name           Name of the remote
  option         Name of the option to modify.
  value          (optional) Value of the option.

optional arguments:
  -h, --help     show this help message and exit
  --global       Use global config.
  --system       Use system config.
  --local        Use local config.
  -q, --quiet    Be quiet.
  -v, --verbose  Be verbose.
  -u, --unset    Unset option.
```

Reply:
`dvc remote modify` is a link to the command in this case (it's a regular way to link commands in docs)

Reply:
The point is that `dvc remote modify` does not provide anything related to S3 API parameters.

Reply:
It does. 

<img width="685" alt="Screen Shot 2020-12-01 at 9 35 13 PM" src="https://user-images.githubusercontent.com/3659196/100832636-1e2e9f80-341d-11eb-92dd-6f88a951d18e.png">

Reply:
Oh, I misunderstood the intention here; didn't realize that it was used as a URL placeholder.